api/integreatly/v1alpha1/grafana_types.go

```diff
@@ -481,6 +481,8 @@ type GrafanaConfigAnalytics struct {
 type GrafanaConfigDashboards struct {
 	// +nullable
 	VersionsToKeep *int `json:"versions_to_keep,omitempty" ini:"versions_to_keep,omitempty"`
+	// +nullable
```

Comment:
I don't think we need a nullable for a non-pointer type, the zero value for a string is `""` (empty string), it would only be an issue if we had a pointer to a string(`*string`), where the zero value would be a `nil`